ga4gh/cli.py

```diff
@@ -797,11 +797,14 @@ class SearchExpressionLevelsRunner(AbstractSearchRunner):
     def __init__(self, args):
         super(SearchExpressionLevelsRunner, self).__init__(args)
         self._rnaQuantificationId = args.rnaQuantificationId
+        if len(args.feature_ids) > 0:
```

Comment:
nit: we usually do this like 
```
self._feature_ids = []
if len(args.feature_ids) > 0:
    self._feature_ids = args.feature_ids.split(",")
```
Cuts down on one line and ensures something is assigned to the attribute, even if the things in the if/else clauses change